### PR TITLE
Add is_in_battle and EX Gyms

### PR DESCRIPTION
**Pull request by zaeshu**
RocketMap:
1. is_in_battle is updated in the DB for Gyms.
2. is_ex_raid_eligible is updated from raids and gyms - Webhook is sent.
3. I removed the weather_boosted_value because the raid_mon is only updated after encountering
This requires a DB modification:
ALTER TABLE 'gym'
ADD 'park' tinyint(1) NULL;

### CODE REVIEW

File: db/rmWrapper.py [hunk 1]
```diff
@@ -445,7 +447,7 @@ def get_near_gyms(self, lat, lng, hash, raid_no, dist, unique_hash="123"):
             ") "
             "AS distance, latitude, longitude, name, description, url "
             "FROM gym "
-            "LEFT JOIN gymdetails ON gym.gym_id = gymdetails.gym_id "
+            "LEFT JOIN gym_details ON gym.gym_id = gym_details.gym_id "
```
Comment (TiMXL73) [inline db/rmWrapper.py]: Shouldn't this be gymdetails and not gym_details for RM's database?

```suggestion
            "LEFT JOIN gymdetails ON gym.gym_id = gymdetails.gym_id "
```

Comment (zaeshu) [reply db/rmWrapper.py]: That isn't part of my commit. Not sure why that's in there. Snoopy fixed it last commit

File: db/rmWrapper.py [hunk 2]
```diff
@@ -170,11 +170,12 @@ def submit_raid(self, gym, pkm, lvl, start, end, type, raid_no, capture_time, un
                 "UPDATE raid "
                 "SET level = %s, spawn = FROM_UNIXTIME(%s), start = %s, end = %s, "
                 "pokemon_id = %s, last_scanned = FROM_UNIXTIME(%s), cp = %s, "
-                "move_1 = %s, move_2 = %s "
```
Comment (sn0opy) [inline db/rmWrapper.py]: You removed this and a few more spaces which are actually required and things will break without them.

**Comment by zaeshu** [comment]
Closing due to rework of webhooks

**Pull request closed without merging**